pg_store_plans.c

```diff
@@ -1120,8 +1120,13 @@ pgsp_ProcessUtility(PlannedStmt *pstmt, const char *queryString,
 					DestReceiver *dest, COMPTAG_TYPE *completionTag)
 {
 	int			tag = nodeTag(pstmt->utilityStmt);
-	queryid_t	saved_queryId = pstmt->queryId;
 	bool		reset_force_disabled = false;
+#if PG_VERSION_NUM >= 140000
```

Comment:
actually `saved_queryId` is only used in the `if (pgsp_enabled(saved_queryId)` part and the source info isn't modified, so we could instead just directly pass `pstmt->queryId` to avoid too much code complication.